bridgestyle/arcgis/togeostyler.py

```diff
@@ -588,10 +589,15 @@ def processSymbolLayer(layer, symboltype, options):
 
     elif layer["type"] == "CIMHatchFill":
         rotation = layer.get("rotation", 0)
-        size = _ptToPxProp(layer, "separation", 3)
         symbolLayers = layer["lineSymbol"]["symbolLayers"]
         color, width = _extractStroke(symbolLayers)
         wellKnowName = _hatchMarkerForAngle(rotation)
+        if rotation % 90:
+           _warnings.append('Can handle rotation value other than 45, 90, 135,... round to the nearast one.°')
```

Comment:
Should it be "Can not" instead of "Can"?

And what happens if it's not one of the values we want? Just a warning and no rotation? (If so, can we add that info to the warning)

Reply:
No, the nearest value will be taken (35° -> 45°) for instance